diff -u -r motion-3.2.8-orig/ffmpeg.c motion-3.2.8/ffmpeg.c
--- motion-3.2.8-orig/ffmpeg.c	2007-07-18 10:18:05.000000000 +0300
+++ motion-3.2.8/ffmpeg.c	2007-10-06 01:47:49.000000000 +0300
@@ -15,6 +15,8 @@
 #include "ffmpeg.h"
 #include "motion.h"
 
+extern URLProtocol file_protocol;
+
 #if LIBAVCODEC_BUILD > 4680
 /* FFmpeg after build 4680 doesn't have support for mpeg1 videos with 
  * non-standard framerates. Previous builds contained a broken hack 
